pkg/controllers/provisioning/scheduling/scheduler.go

```diff
@@ -220,9 +256,25 @@ func (s *Scheduler) Solve(ctx context.Context, pods []*corev1.Pod) Results {
 		).Set(float64(len(q.pods)))
 
 		if s.clock.Since(lastLogTime) > time.Minute {
-			log.FromContext(ctx).WithValues("pods-scheduled", batchSize-len(q.pods), "pods-remaining", len(q.pods), "duration", s.clock.Since(startTime).Truncate(time.Second), "scheduling-id", string(s.id)).Info("computing pod scheduling...")
+			log.FromContext(ctx).WithValues(
+				"pods-scheduled", batchSize-len(q.pods),
+				"pods-remaining", len(q.pods),
+				"duration", s.clock.Since(startTime).Truncate(time.Second),
+				"scheduling-id", string(s.id),
+			).Info("computing pod scheduling...")
 			lastLogTime = s.clock.Now()
 		}
+
+		if s.timeout != nil && s.clock.Since(startTime) > *s.timeout {
+			log.FromContext(ctx).WithValues(
```

Comment:
Is this more of a debug message more than anything else?

Reply:
Yes, but I felt it was appropriate to keep it at the same debug level as the progress log line. I think there's an argument to drop both of those down to debug, but if you're running a cluster large enough for either of these to be printed, this isn't exactly going to be spammy.